### PR TITLE
Transmit command line args to render clients

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: eq/server/config.h [hunk 1]
```diff
@@ -139,6 +139,12 @@ class Config : public fabric::Config< Server, Config, Observer, Layout,
     /** @return the name of the render client executable. */
     const std::string& getRenderClient() const { return _renderClient; }
 
+    /** Set the render client command line options. */
+    void setRenderClientArgs( const Strings& args ){ _renderClientArgs = args; }
```
Comment (hernando) [inline eq/server/config.h]: Same thing for the environment?

File: eq/server/server.cpp [hunk 2]
```diff
@@ -218,6 +218,7 @@ bool Server::_cmdChooseConfig( co::ICommand& command )
     config->setApplicationNetNode( node );
     config->setWorkDir( params.getWorkDir( ));
```
Comment (hernando) [inline eq/server/server.cpp]: Do you know if this was added after I did the hack to fix file paths?

Comment (eile) [inline eq/server/server.cpp]: version 1.0 says doxygen. Not sure when you added this.

File: eq/server/config.h (hunk 1 above)
Comment (eile) [inline eq/server/config.h]: Not sure yet - might also be a list of env variable patterns to transmit, e.g., 'RTNEURON_*, OSGTRANSPARENCY_*'

**Comment by hernando** [comment]
I'm working on adding the env var. I'll submit a PR to you.

File: eq/fabric/configParams.h [hunk 3]
```diff
@@ -93,6 +93,17 @@ class ConfigParams
     EQFABRIC_API const std::string& getRenderClient() const;
 
     /**
+     * Set the command line arguments for the render client executable.
+     *
+     * If no arguments are provided, eq::Server::chooseConfig() uses
+     * co::LocalNode::getCommandLine().
+     */
+    EQFABRIC_API void setRenderClientArgs( const Strings& args );
```
Comment (hernando) [inline eq/fabric/configParams.h]: setClientArgs?
version number here?

Comment (eile) [inline eq/fabric/configParams.h]: Used for symmetry with setRenderClient().

File: eq/server.cpp [hunk 4]
```diff
@@ -88,6 +88,8 @@ Config* Server::chooseConfig( const fabric::ConfigParams& p )
         params.setWorkDir( Global::getWorkDir( ));
     if( params.getRenderClient().empty( ))
         params.setRenderClient( Global::getProgramName( ));
+    if( params.getRenderClientArgs().empty( ))
+        params.setRenderClientArgs( client->getCommandLine( ));
     if( params.getGPUFilter().empty( ))
```
Comment (tribal-tec) [inline eq/server.cpp]: setRenderClientEnvPrefixes missing here?

**Comment by tribal-tec** [comment]
+1 otherwise

File: eq/server.cpp (hunk 4 above)
Comment (eile) [inline eq/server.cpp]: Not needed - what would you provide here?

**Pull request merged**